### PR TITLE
Enhancements: Add comments in tos.ts and switch Twitter icon to X

### PR DESCRIPTION
**Pull request by NikolaiKryshnev**
Included comments explaining the purpose of the array and the meaning of each code.

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/bridge/pages/api/tos.ts [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
+  'BG', // Bulgaria
+  'CY', // Cyprus
+  'CZ', // Czech Republic
+  'DE', // Germany
+  'DK', // Denmark
+  'EE', // Estonia
+  'ES', // Spain
+  'FI', // Finland
+  'FR', // France
+  'GB', // United Kingdom (although it left the EU)
```
Comment (wbnns) [inline apps/bridge/pages/api/tos.ts:16]: Please drop the parenthetical

**Review by wbnns — DISMISSED** [review]
@NikolaiKryshnev 

Thanks! Left a small in-line nit

**Review by zencephalon — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for zencephalon @ 2024-11-07 20:42:32 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by zencephalon — APPROVED** [review]

**Pull request merged**